File: GTTS/INL.TrainingService.Models/ISaveTrainingEventRoster_Param.cs
using System.IO;
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public interface ISaveTrainingEventRoster_Param
    {
        long TrainingEventID { get; set; }
        long? TrainingEventGroupID { get; set; }
        string ParticipantType { get; set; }
        List<ITrainingEventRoster_Item> Participants { get; set; }
        Stream StudentExcelStream { get; set; }
        int? ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/isave-training-event-roster_param.ts



import { ITrainingEventRoster_Item } from './itraining-event-roster_item';

export interface ISaveTrainingEventRoster_Param {
  
	TrainingEventID: number;
	TrainingEventGroupID?: number;
	ParticipantType: string;
	Participants?: ITrainingEventRoster_Item[];
	StudentExcelStream?: any;
}





